src/stores/likes.js

```diff
@@ -26,6 +29,9 @@ export const useLikeStore = defineStore('likes', {
 
       this._likes = likesSnapshot.docs.map((doc) => doc.data())
       this._dislikes = dislikesSnapshot.docs.map((doc) => doc.data())
+
+      this._isLoading=false
+      this._isLoaded=true
```

Comment:
Seems redundant to me.

In my opinion, if it **is not loading** this means it **is loaded**.

Reply:
Sorry @arnonrdp i had an internet connection cut when i was replying to this. 

Both are needed. 

The loading state is initially set to false, which means that the content is not loaded yet. However, we want to perform some action only when the content is fully loaded and ready to use.
Therefore, we cannot rely on the loading state alone to determine when to execute the action. If we do that, we might run into an error because the action might depend on some data that is not available yet. 
To avoid this problem, we need to use another variable or condition that indicates when the content is loaded and the action can be safely performed.

Reply:
Now I got it. Makes sense.